input/europe/eu.csv

```diff
@@ -0,0 +1,150 @@
+Country,Name,Reactor no.,Reactor,Status,Capacity in MW,Construction start date,Commercial operation date,Closure (planned),,
```

Comment:
Question: Did you enter this information manually? Where did it come from? If you pulled it from the internet with a script (or if you could) it could be easier to update someday if that script were included here.